src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -29,173 +32,96 @@ import { TradingRewardsSummaryPanel } from './TradingRewardsSummaryPanel';
 
 const RewardsPage = () => {
   const stringGetter = useStringGetter();
-  const { isTablet, isNotTablet } = useBreakpoints();
   const navigate = useNavigate();
 
+  const { complianceState } = useComplianceState();
+  const { isTablet, isNotTablet } = useBreakpoints();
+
   const { enableStaking, tradingRewardsRehaul: tradingRewardsRehaulEnabled } = testFlags;
 
+  const showMigratePanel = import.meta.env.VITE_V3_TOKEN_ADDRESS && isNotTablet;
+  const showGeoblockedPanel =
+    tradingRewardsRehaulEnabled && complianceState !== ComplianceStates.FULL_ACCESS;
```

Comment:
how do i get to this state again? i tried turning on "simulate restricted geo" but couldn't see it :<

Reply:
oh "simulate restricted geo" doesn't work outside of mainnet atm 😅 You can simulate in the config with setting account to BLOCKED